core/domain/stats_domain.py

```diff
@@ -744,6 +744,9 @@ def __init__(
     def to_dict(self):
         """Returns a dict representation of the Playthrough domain object.
 
+        The keys of the dict **must** correspond to the properties of the
```

Comment:
Can this be asserted in a backend test to avoid regressions?

Reply:
Not easily, so I've just explicitly assigned to each property instead.